design/adr/core/0002-Developer-Scripts-TaggingAndCleanup.md

```diff
@@ -0,0 +1,105 @@
+# Developers Script - Clearing and Tagging
+
+<!--ts-->
+
+- [Status](#status)
+- [Context](#context)
+- [Proposed Design](#proposed-design)
+- [Decision](#decision)
+- [Consequences](#consequences)
+- [References](#references)
+
+<!--te-->
+
+## Status
+
+Proposed
+
+## Context  
+
+`Developer Script` repository contains helper scripts used by the EdgeX developers in their everyday work, docker-compose files for all the releases, and scripts for gathering project metrics. <br />
+This repository have only one branch - `master`. 
```

Comment:
Quick note if we go the route of versioning developer scripts with the CI then we will only be able to track one version for developer scripts.